packages/notary/actors/notary_actor.py

```diff
@@ -38,21 +54,29 @@ def get(self, key, schema_out):
         ```
         """
         out = schema_out.new()
-        # TODO: return json object
-        out.message = "your key is {}".format(key)
+        bcdb = self.bcdb_get()
+        for model in bcdb.get_all():
+            if model.key == key:
+                return model
+
+        out.message = "this key doesn't exist"
         return out
 
-    def delete(self, robot_id, key, key_signature, schema_out):
+    def delete(self, threebot_id, key, content_signature, schema_out):
```

Comment:
We do no need a delete method after all, see comments: https://github.com/threefoldtech/home/issues/135#issuecomment-468336892